compute/oslogin/service_account_ssh.py

```diff
@@ -97,17 +97,17 @@ def run_ssh(cmd, private_key_file, username, hostname):
 
 
 # [START main]
-def main(
-        cmd, project, instance, zone, oslogin=None, account=None, hostname=None
-        ):
+def main(cmd, project, instance=None, zone=None,
+         oslogin=None, account=None, hostname=None):
```

Comment:
oslogin is not provided by the call from the body below.

Reply:
It shouldn't be necessary unless your application is initializing the API manually. service_account_ssh_test.py does this because it cannot manage SSH keys on the behalf of another service account, so it has to obtain the API key for the service account and initialize the API manually. If you don't provide an oslogin object, main() will generate one for you using whatever application default credentials it can obtain from the instance or environment variables.